projects/components/src/summary-box/summary-box.component.ts

```diff
@@ -0,0 +1,55 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { Color } from '@hypertrace/common';
+
+@Component({
+  selector: 'ht-summary-box',
```

Comment:
This sounds very similar to summary-value component.

Reply:
refactored to x-more